README.md

```diff
@@ -1,4 +1,47 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
+
+# DEREK_SONG . Jarvis Consulting
```

Comment:
Derek Song

Reply:
Hello Edward, I have created a new branch feature/resume2 upon request, and corrected all the mistakes. A new pull request is posted in my 360 page. Thank you so much.